cmd/process-agent/app/events.go

```diff
@@ -35,11 +43,20 @@ var EventsListenCmd = &cobra.Command{
 	SilenceUsage: true,
 }
 
+// EventsPullCmd is a command to pull process lifecycle events
+var EventsPullCmd = &cobra.Command{
+	Use:          "pull",
+	Short:        "Periodically pull process lifecycle events. This feature is currently in alpha version and needs root privilege to run.",
+	RunE:         runEventStore,
+	SilenceUsage: true,
+}
+
 func init() {
-	EventsCmd.AddCommand(EventsListenCmd)
+	EventsCmd.AddCommand(EventsListenCmd, EventsPullCmd)
+	EventsPullCmd.Flags().DurationVarP(&pullInterval, "tick", "t", 5*time.Second, "The period between 2 consecutive pulls to fetch process events")
```

Comment:
Let's separate the `pullInterval` from the default 5 seconds. This looks like a good pattern to follow:
https://github.com/DataDog/datadog-agent/blob/906dd6798d41aef875cb7c60899233619b762176/cmd/agent/app/dogstatsd_capture.go#L41

Reply:
Thanks for pointing this pattern. It's indeed much better than the previous solution.
I've refactored it in https://github.com/DataDog/datadog-agent/pull/12159/commits/5f95d06ed2feb6fecd77eb0f7c7086893b3f5b50.